benchmarks/postgres-kafka-cdc/benchi.yml

```
@@ -0,0 +1,76 @@
+infrastructure:
+  postgres:
+    compose: ../../shared/postgres/compose.yml
+  kafka:
+    compose: ../../shared/kafka/compose.yml
+
+tools:
+  kafka-connect:
+    compose:
+      - ../../shared/kafka-connect/compose.yml
+      - ./kafka-connect/compose.override.yml
+  conduit:
+    compose:
+      - ../../shared/conduit/compose.yml
+      - ./conduit/compose.override.yml
+
+metrics:
+  conduit_metrics:
+    collector: conduit
+    tools: conduit
+    settings:
+      url: http://localhost:8080/metrics
+      pipelines: postgres-to-kafka
+  kafka_metrics:
+    collector: kafka      
+    settings:
+      url: http://localhost:7071/metrics
+      topics: postgres.public.employees
+  docker_metrics:
+    collector: docker
+    settings:
+      containers:
+        - benchi-conduit
+        - benchi-kafka-connect
+tests:
+  - name: pg-to-kafka-cdc
```

Let's keep the name of the test the same as the folder it's in.
```suggestion
  - name: postgres-kafka-cdc
```